portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -43,10 +48,13 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
 
-    List<String> comments = new ArrayList<String>();
+    HashMap<String, ArrayList<String>> comments = new HashMap<String, ArrayList<String>>();
     for (Entity entity : results.asIterable()) {
-      String comment = (String) entity.getProperty("comment");
-      comments.add(comment);
+      String id = KeyFactory.keyToString(entity.getKey());
+      ArrayList<String> commentEmail = new ArrayList<String>();
+      commentEmail.add((String) entity.getProperty("comment"));
+      commentEmail.add((String) entity.getProperty("userEmail"));
+      comments.put(id, commentEmail);
     }
```

Comment:
Indentation should be 2 spaces as per [here](https://google.github.io/styleguide/javaguide.html#s4.2-block-indentation)

Reply:
This indentation is 2 spaces

Reply:
Did you run clang-format?

Reply:
Yes I did